src/tests/suite/daos_test.c

```diff
@@ -368,7 +368,7 @@ main(int argc, char **argv)
 	memset(tests, 0, sizeof(tests));
 
 	while ((opt = getopt_long(argc, argv,
-				  "ampcCdtTVizxADKeoROg:n:s:u:E:f:Fw:W:hrNvbSl:",
+				"ampcCdtTVizUxADKeoROg:n:s:u:E:f:Fw:W:hrNvbSl:",
```

Comment:
```suggestion
				  "ampcCdtTVizUxADKeoROg:n:s:u:E:f:Fw:W:hrNvbSl:",
```

Reply:
Going to ignore this change request. It makes the line longer than 80